### ISSUE TITLE
Distinguish between first destroy that softdeletes and second destroy that actually deletes

### ISSUE THREAD

**Issue by thebravoman** [ActsAsParanoid/acts_as_paranoid#169]
Hi, 

I can not find a way to distinguish between the first destroy that only sets the delete_at and the second destroy that deletes the file. I need this to destroy and external file on the second destroy and I need to understand when to do it. 

In an after_destroy I can not use Dirty changes. They do not tell us that deleted_at as changed. 

If there is a way I would suggest to add this to the documentation. If there is no way I think this would be a great addition.

### Update 1
Found out that the deleted_at attribute is cleard
```
# acts_as_paranoid-0.6.3/lib/acts_as_paranoid/core.rb:291
   def stale_paranoid_value
      self.paranoid_value = self.class.delete_now_value
      clear_attribute_changes([self.class.paranoid_column])
    end
````

Kind of makes sense. The change of the paranoid_column is hidden. Continue looking for a way to say if it is completed deleted or softdeleted.

### Update 2 - Found it

There is a method "destroyed_fully?" and "deleted_fully?" that will distinguish between the two. 

Would you like me to prepare a new PR for the readme that adds this info there. I would have to add this section in our inside acts_as_paranoid readme for all the colleagues, but I think it would be better for it to be here?

**Comment by mvz** [comment]
> Would you like me to prepare a new PR for the readme that adds this info there. I would have to add this section in our inside acts_as_paranoid readme for all the colleagues, but I think it would be better for it to be here?

Yes, that would be great!

**Comment by thebravoman** [comment]
I've added it to the status section #170

**Comment by mvz** [comment]
Fixed in #170!
